Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -191,6 +193,21 @@ CommandLineArgs::CommandLineArgs(const std::vector<std::wstring> &args)
                 PrintUsage();
                 throw hresult_invalid_argument(L"Unknown Mode!");
             }
+            try
```

Comment:
On line 9, can we change the usage to:

-SaveTensorData <saveMode> <optional: folder path>: save first iteration or all iteration output tensor results to csv file [First, All]". Optional folder path can be specified to hold tensor data. It will be created if folder doesn't exist.